build.gradle.kts

```diff
@@ -1,8 +1,8 @@
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
```

Comment:
@fabian-hk the changes here aim to use java 17 since I faced compatibility issues when integrating the lib in a Springboot 3.x project (only java 17 can be used with that version of Springboot)
More details on that can be found here:
https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.0-Release-Notes